File: app/src/main/java/com/jedev/leflix/service/API/Entities/SaleInfo.java
package com.jedev.leflix.service.API.Entities;

import java.util.List;

public class SaleInfo {
    public String country;
    public String saleability;
    public boolean isEbook;
    public ListPrice listPrice;
    public RetailPrice retailPrice;
    public String buyLink;
    public List<Offer> offers;
}

File: app/src/main/java/com/jedev/leflix/service/api/entities/SaleInfo.kt
package com.jedev.leflix.service.api.entities

data class SaleInfo(var country: String,
                    var saleability: String,
                    var isEbook: Boolean,
                    var listPrice: ListPrice,
                    var retailPrice: RetailPrice,
                    var buyLink: String,
                    var offers: List<Offer>)
